spring-xd-dirt/src/main/java/org/springframework/xd/dirt/stream/AbstractDeployer.java

```diff
@@ -24,6 +24,8 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
+import org.apache.curator.framework.CuratorFramework;
+import org.apache.curator.framework.api.BackgroundPathAndBytesable;
```

Comment:
yeah, Initially I had the migration code in here and then moved to `StreamDeployer`. Fixing this now.